server/src/Ogmios/Parser.hs

```diff
@@ -0,0 +1,146 @@
+module Ogmios.Parser (decodeProtocolParameters) where
+
+import Cardano.Api (
+  AnyPlutusScriptVersion (AnyPlutusScriptVersion),
+  CostModel,
+  ExecutionUnitPrices (ExecutionUnitPrices),
+  ExecutionUnits,
+  Lovelace (Lovelace),
+  PlutusScriptVersion (PlutusScriptV1),
+ )
+import Cardano.Api.Shelley (ProtocolParameters (ProtocolParameters))
+
```

Comment:
Why the blank lines between imports? Fourmolu will sort the imports if you remove them

Reply:
I separate Imports by category : 

- Cardano/Plutus related
- Base Haskell modules 
- Rest of the packages 
- If some package has a lot of imports, then it has its own section 

Those can be in random order but are always separated. This helps me to maintain imports and looks better (to me) than having all of them together. I can change those for consistency or change the rest of the server to this format.

Reply:
Just for consistency with the rest of the codebase (both Haskell and Purescript), I'd prefer if you don't separate them